### PR TITLE
Update documentation with mandatory service and subservice

### LINKED ISSUE #352: Update the documentation to reflect the changes in function signatures
With the last refactor, most of the IOTA function signatures where changed to add the service and subservice parameters to the call, but this change was not reflected in the documentation.

### PR DESCRIPTION
**Pull request by dmoranj**
Fixes #352

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -443,59 +449,72 @@ The handler is expected to call its callback once with no parameters (failing to
 function setConfigurationHandler(newHandler)
 ```
 ###### Description
-Sets the new user handler for the configuration updates. This handler will be called every time a new configuration is created or an old configuration is updated. 
+Sets the new user handler for the configuration updates. This handler will be called every time a new configuration is
+created or an old configuration is updated.
 
 The handler must adhere to the following signature:
 ```
 function(newConfiguration, callback)
```
Comment (gtorodelvalle) [inline README.md]: I guess the name for this function is missing... :)

Comment (dmoranj) [inline README.md]: Well, it's not missing, really. That's the prototype for the handler function the user have to pass to the `setConfigurationHandler()` function being described in that section. The user can use any name they want.

Comment (gtorodelvalle) [inline README.md]: NTC

**Comment by gtorodelvalle** [comment]
LGTM

**Pull request merged**